.buildkite/scripts/steps/run-agentless-tests.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+
+# ELASTICSEARCH CONFIDENTIAL
+# __________________
+#
+#  Copyright Elasticsearch B.V. All rights reserved.
+#
+# NOTICE:  All information contained herein is, and remains
+# the property of Elasticsearch B.V. and its suppliers, if any.
+# The intellectual and technical concepts contained herein
+# are proprietary to Elasticsearch B.V. and its suppliers and
+# may be covered by U.S. and Foreign Patents, patents in
+# process, and are protected by trade secret or copyright
+# law.  Dissemination of this information or reproduction of
+# this material is strictly forbidden unless prior written
+# permission is obtained from Elasticsearch B.V.
+
+set -eox pipefail
+
+_SELF=$(dirname $0)
+source "${_SELF}/../common.sh"
+
+extract_sha() {
+    local env=$1
+    local sha
+
+    # Ensure repo is available
+    git clone --depth 1 git@github.com:elastic/serverless-gitops.git || (
```

Comment:
https://github.com/elastic/serverless-gitops is a private github repository. Do we know if this will work? Or does it require some specific settings to work?

Reply:
Yes, need to merge [this](https://github.com/elastic/terrazzo/pull/369) before